public_html/wp-content/mu-plugins/blocks/assets/src/shared/image-alignment/index.js

```diff
@@ -38,6 +38,7 @@ function ImageAlignmentControl( {
 				{ options.align.map( ( alignment ) => {
 					const optLabel = alignment.label;
 					const optValue = alignment.value;
+					// should we use `const { label, value } = alignment;` here for consistency?
```

Comment:
I didn't do that in this case because `label` and `value` are already used in the component signature.

Reply:
Ah, ok, that makes sense.

I wonder if we need the extra variable, though? Would it be slightly more straight-forward to  just `alignment.label`?